assets/js/components/user-input/UserInputKeywords.js

```diff
@@ -121,7 +127,9 @@ export default function UserInputKeywords( { slug, max } ) {
 						} ) }
 					>
 						<TextField
-							label={ i + 1 === values.length ? __( 'Enter minimum one (1), maximum three (3) terms', 'google-site-kit' ) : '' }
+							label={ values.length === 1 && values[ 0 ] === ''
+								? __( 'Enter minimum one (1), maximum three (3) terms', 'google-site-kit' )
+								: '' }
```

Comment:
@tofumatt do we really need to hide this placeholder? I think it's quite helpful as is especially after all modifications in the keywords component.

Reply:
The text field becomes invisible, so I think we need to revert this. 
![google github io_site-kit-wp_storybook_pull_2984__path=_story_user-input--userinputapp](https://user-images.githubusercontent.com/873049/112021538-1284f700-8b3a-11eb-8664-53dbf56de673.png)


https://google.github.io/site-kit-wp/storybook/pull/2984/?path=/story/user-input--userinputapp

Reply:
Oh, agreed actually. Especially now that the input often spans multiple lines the removal of both the placeholder _and_ the blue underline makes it look both spartan and unclear.

I mean, the original issue was when everything was on one line, eg:

<img width="731" alt="Screen Shot 2021-02-17 at 8 44 00 PM" src="https://user-images.githubusercontent.com/373931/109562970-f985aa80-7a93-11eb-8ea5-999537162152.png">

Maybe now it's best if we leave this alone and mark the bug as won't fix.

Thoughts @felixarntz and @ivankruchkoff (as the original issue creator)?